packages/runtime/src/useCases/common/RuntimeErrors.ts

```diff
@@ -193,6 +193,13 @@ class Attributes {
             `Attribute '${attributeId.toString()}' is not a third party owned relationship attribute.`
         );
     }
+
+    public cannotSeparatelyDeleteChildOfComplexAttribute(attributeId: CoreId | string): ApplicationError {
```

Comment:
Complex Attributes are always IdentityAttributes, so I think it is sufficient to use the shorter version. This is also consistent with the naming in the AttributesController.

Reply:
Since we only talk about complex IdentityAttributes in the documentation (https://enmeshed.eu/integrate/attribute-introduction#complex-identityattributes) and error messages are both visible to Integrators and documented, unlike methods of the AttributesController, I would be in favor of the more precise name for the error name to avoid confusion. However, it is certainly possible to hold a different opinion, which is why I have no problem leaving the decision up to you.